test/functional/test_framework/p2p.py

```
@@ -588,13 +589,13 @@ def on_inv(self, message):
     def on_ping(self, message):
         self.send_message(msg_pong(message.nonce))
 
-    def on_mnlistdiff(self, message): pass
     def on_clsig(self, message): pass
-    def on_islock(self, message): pass
     def on_isdlock(self, message): pass
-
-    def on_qgetdata(self, message): pass
+    def on_islock(self, message): pass
```

Flake8 (7.2.0)

[error] multiple statements on one line (def)

(E704)